pkg/scope/machine.go

```diff
@@ -104,6 +105,36 @@ func (m *MachineScope) PatchObject(ctx context.Context) error {
 	return m.patchHelper.Patch(ctx, m.HivelocityMachine)
 }
 
+// DeviceTagMachine returns a DeviceTag object for the machine tag.
+func (m *MachineScope) DeviceTagMachine() hvtag.DeviceTag {
+	return hvtag.DeviceTag{
+		Key:   hvtag.DeviceTagKeyMachine,
+		Value: m.Name(),
+	}
+}
+
+// DeviceTagCluster returns a DeviceTag object for the cluster tag.
+func (m *MachineScope) DeviceTagCluster() hvtag.DeviceTag {
```

Comment:
This method could be moved to DeviceScope `NewDeviceTag()`. But not merge-blocking.

Reply:
Moved the method - it is now defined as HivelocityCluster.DeviceTag()